File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/MathQuestion1IncorrectAnswer.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class MathQuestion1IncorrectAnswer : StoryChoice
	{
		internal MathQuestion1IncorrectAnswer() : base(StoryChoice.GetLocalizedString("ch5MathQuestion1IncorrectAnswer"), 5048)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5AnsweredMathQuestion1 = TwoChoiceAnswer.Incorrect;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/MathQuestion1IncorrectAnswer.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.TwoChoiceAnswer;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class MathQuestion1IncorrectAnswer extends StoryChoice {
    public MathQuestion1IncorrectAnswer() {
        super(R.string.ch5MathQuestion1IncorrectAnswer, 5048);
    }

    @Override
    public void select()
    {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5AnsweredMathQuestion1(TwoChoiceAnswer.Incorrect);
        super.select();
    }
}
